Alisha/data/lab1.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    import matplotlib.pyplot as plt
</pre><p>Always add imports at top cell of the file </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>